sdcm/cluster.py

```diff
@@ -2394,7 +2394,7 @@ def upgrade_mgmt(self, scylla_mgmt_address, start_manager_after_upgrade=True):
             time.sleep(5)
 
     # pylint: disable=too-many-branches
-    def install_mgmt(self, package_url: Optional[str] = None) -> None:
+    def install_mgmt(self, package_url: Optional[str] = None, force_package_name=None) -> None:
```

Comment:
i think it is better to change it to be something like `package_names` and then on line 2416 you can check:
```
if not package_names:
    package_names = "scylla-manager-server scylla-manager-client"
        if package_url:
            package_names = f"{package_url}scylla-manager-server* {package_url}scylla-manager-client*"
        elif self.parent_cluster.params.get("scylla_mgmt_address"):
            self.download_scylla_manager_repo(self.parent_cluster.params.get("scylla_mgmt_address"))
        else:
            manager_version = self.parent_cluster.params.get("manager_version")
            manager_repo_url = get_manager_repo_from_defaults(manager_version, self.distro)
            self.download_scylla_manager_repo(manager_repo_url)
```

Reply:
Sure

Reply:
But if I do that, when `package_names` won't be `None`, the packages won't be downloaded.